tests/test_template_types.py

```diff
@@ -105,13 +130,13 @@ def test_markdown_in_templates(template_class):
             'subject': 'animal story'
         },
         {'animal': 'fox', 'colour': 'brown'}
-    )
-    assert (
-        '<p style="Margin: 0 0 20px 0; font-size: 19px; line-height: 25px; color: #0B0C0C;">'
-        'the quick brown fox</p>'
-        '<p style="Margin: 0 0 20px 0; font-size: 19px; line-height: 25px; color: #0B0C0C;">'
-        'jumped over the lazy dog</p>'
-    ) in str(template)
+    ))
+    if template_class == HTMLEmailTemplate:
+        mock_notify_email_markdown.assert_called_once_with(result)
+    elif template_class == LetterPreviewTemplate:
+        mock_notify_letter_preview_markdown.assert_called_once_with(result)
+    else:
+        assert False
```

Comment:
i really don't like `if` statements in tests - you could pass in the mock string as a parametrise option and then write something like

```
def test_markdown_in_templates(template_class, result, markdown_render):
    with mock.patch(markdown_renderer) as mock:
        str(template) [...]
    mock.assert_called_once_with(result)
```

that saves us from the icky `else: assert False` line too